### ISSUE TITLE
Picker's title accessibilityRole is missing

### ISSUE THREAD

**Issue by yaldaafshar** [wix/react-native-ui-lib#2801]
## Description
In the Picker component, in topBarProps, there is no accessibilityRole or in general accessibilityProps. This causes ADA bugs when someone tries to make the title more meaningful for people with diasbilities. 

### Related to

- [ ] Components
- [ ] ADA

### Steps to reproduce

1. Go to picker component, check topBarProps accepting type which is ModalTopBarProps, there is no accessibilityProps defined for title. 
2. Turn on screen reader where picker with title is used, it is not announcing title role as 'heading'

#### Expected behavior

- React Native UI Lib version: 7.7.0

### Affected platforms

- [ ] Android
- [ ] iOS
